### ISSUE TITLE
Editor config options ignored in setup

### ISSUE THREAD

**Issue by shtrom** [asamuzaK/withExEditorHost#52]
For some reason, the `--editor-path` `--overwrite-editor-config` and `--editor-args` seem to be ignored by the `setup` script on `master`. Oddly enough, `--overwrite-config` and `--browser` are fine:

```
$ npm run setup -- --overwrite-config --browser=firefox --editor-path $(which gvim) --overwrite-editor-config --editor-args ""
Current branch master is up to date.

> withexeditorhost@5.0.0-a.2 setup /Users/omehani/bordel/default-env/vendor/withExEditorHost
> npm install --production && node ./index.js setup "--overwrite-config" "--browser=firefox" "--editor-path" "/usr/local/bin/gvim" "--overwrite-editor-config" "--editor-args" ""

audited 2934 packages in 2.484s
found 0 vulnerabilities

(node:83828) ExperimentalWarning: The fs.promises API is experimental
Created: /Users/omehani/Library/Application Support/withexeditorhost/config/firefox
Created: /Users/omehani/Library/Application Support/withexeditorhost/config/firefox/withexeditorhost.sh
Created: /Users/omehani/Library/Application Support/Mozilla/NativeMessagingHosts/withexeditorhost.json
/Users/omehani/Library/Application Support/withexeditorhost/config/firefox/editorconfig.json already exists.
Overwrite? [y/n]: y
Input editor path: /usr/local/bin/gvim
Input command line options:
Created: /Users/omehani/Library/Application Support/withexeditorhost/config/firefox/editorconfig.json
```

I would have expected everything from the `Overwrite` prompt to be pulled from the options in the command line.

It looks like a resurgence of #41, though it even ignores non-empty arguments that are provided. Similarly, passing a dummy non-empty argument for `--editor-args` doesn't fix the issue.

**Comment by asamuzaK** [comment]
Confirmed.
Thanks for your report.

**Comment by asamuzaK** [comment]
Released v5.0.0-b1 [Releases · asamuzaK/withExEditorHost](https://github.com/asamuzaK/withExEditorHost/releases)
